changelog/fragments/operator-name-config-file.yaml

```diff
@@ -0,0 +1,10 @@
+entries:
+  - description: >
+      Added `projectName` key to the PROJECT config file (v3-alpha+).
+    kind: addition
+    breaking: false
+    migration:
+      header: Add the `projectName` config key to your PROJECT file
+      body: >
+        Set the `projectName` key in your PROJECT file. If this key is not set,
```

Comment:
👍 have the flag as suggested with projectName was blocker here that is solved now